README.md

```diff
@@ -147,6 +170,22 @@ _Example_:
     }
 ```
 
+_Example in Hiera_:
+
+```yaml
+firewalld::custom_service:
```

Comment:
As per my comment in #52 you seem to have a mixture of custom_service and custom_services going on.....